File: problems/problem01/problem01.c
#include "utility.h"

int search(int* a, int sz, int i) {
    /* fill me in */
    return 0;
}

int main() {
    int sz;
    int* a = read_ints(stdin, &sz);
    printf("%d\n", search(a, sz - 1, a[sz - 1]));
}

File: problems/problem01/problem01.js
function search(a, i) {
    /* fill me in */
}



function main() {
    let fs = require('fs');
    let stdin = fs.readFileSync(0).toString();
    let lines = stdin.trim().split('\n');
    let numbers = lines.map(x => parseInt(x));
    let search_value = numbers.pop();
    let result = search(numbers, search_value);
    console.log(result);
}

main();
